core-services/prow/02_config/_config.yaml

```diff
@@ -283,7 +283,7 @@ plank:
         mediaTypes:
           log: text/plain
       gcs_credentials_secret: gce-sa-credentials-gcs-qe-private-deck
-      timeout: 22h0m0s
+      timeout: 23h0m0s
```

Comment:
Why are you changing this file? Is this intentional or the branch requires rebase? It will require approval from DPTP.

Reply:
Hmm, this is strange. I was just merging the upstream master, so this isn't a change done by myself.

Reply:
I forgot to run `make update` after the merge. Maybe this is a cause?!?

Reply:
Nope.

IDK, why I'm getting this file. Maybe I'll just merge, my changes onto a single commit and put it on top of upstream. Strange, indeed.

Thanks for catching this @mgencur.

Reply:
I squashed the commits into 56e528dee7a9e1d315ca5c00099231510aee4f21